nameko/constants.py

```diff
@@ -1,5 +1,5 @@
 AMQP_URI_CONFIG_KEY = 'AMQP_URI'
-WEB_SERVER_CONFIG_KEY = 'WEB_SERVER_ADDRESS'
+WEB_SERVER_CONFIG_KEY = 'WEB_SERVER_URI'
```

Comment:
I don't break backcompat. I just changed the default config key to the new value.
In the code that interprets this I check for the old key.